packages/api/src/models/farmExternalIntegrationsModel.js

```diff
@@ -92,6 +92,13 @@ class FarmExternalIntegrations extends Model {
       .patch({ webhook_address: webhookAddress, webhook_id: webhookId })
       .where('farm_id', farmId);
   }
+
+  static async getOrganizationId(farmId, partnerId) {
+    return FarmExternalIntegrations.query()
+      .select('organization_uuid')
+      .where('farm_id', farmId)
+      .where('partner_id', partnerId);
```

Comment:
This will allow you to get only the element, and not the element within an array

```suggestion
    return FarmExternalIntegrations.query()
      .select('organization_uuid')
      .where('farm_id', farmId)
      .where('partner_id', partnerId)
      .first();
```